modules/customizing-the-web-console-URL.adoc

```diff
@@ -67,3 +72,75 @@ spec:
     secret:
       name: console-tls
 ----
+
+[id="customizing-the-download-route_{context}"]
+== Customizing the `download` route
+
+.Procedure
+
+. Modify the cluster instance created by default during installation in the `route.openshift.io` custom resource:
++
+[source,terminal]
+----
+$ oc patch route.openshift.io cluster --patch '{"spec":{"route":{"hostname":"console.example.com"}}}' --type=merge
+----
++
+[source,yaml]
+----
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  namespace: openshift-console
+  name: downloads-custom
+spec:
+  tls:
+    termination: edge
+    insecureEdgeTerminationPolicy: Redirect
+  port:
+    targetPort: http
+  to:
+    kind: Service
+    name: downloads
+  wildcardPolicy: None
+----
+. If you specify a custom certificate, you must create a secret in the `openshift-config` namespace that has the key and certificate. For example:
++
+[source,terminal]
+----
+$ oc create secret tls console-tls --key=key.pem --cert=cert.pem -n openshift-config
+----
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: console-tls
+  namespace: openshift-config
+type: kubernetes.io/tls
+data:
+  tls.crt: >-
+        <base64-encoded_cert> ... <1>
+  tls.key: >-
+        <base64-encoded_key> ... <2>
+----
+<1> Provide a valid TLS certificate.
+<2> Provide a valid TLS key.
++
+Then, edit the web console’s Operator configuration:
++
+[source,terminal]
+----
+$ oc edit consoles.operator.openshift.io cluster
+----
++
+Add the following stanza to the configuration resource:
```

Comment:
This block describes the deprecated workflow, thats setting the custom route on the `console-operator`.